low-level/sensors/impl/generic/node_hw_alerts_info.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2001-2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+from collections import namedtuple
+from framework.base.sspl_constants import DEFAULT_RECOMMENDATION
+
+Alert = namedtuple('AlertInfo',
```

Comment:
why this whole code in separate file, can stay with relevant sensor code !?

Reply:
Since we have description, impact, and recommendation along with alert_type and severity for each event for each hw type, this static data is growing huge, for Disk it is still very small but for PSU (100 lines) and other hw types, there are lots of events, so I thought of separating this static data.
Static data can be stored in the following way =>
```python3
alert_for_event = {
    "Drive Slot / Bay": {
           "Event Occured" : {
                  "Assertion": Alert(),
                  "Deassertion": Alert()
            },
            ...
     },
    "Power Supply": {},
    "Temperature": {},
    "Fan": {},
     ...
}
```

Reply:
ok makes sense